packages/core-flows/src/stock-location/steps/update-stock-locations.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  FilterableStockLocationProps,
+  IStockLocationServiceNext,
+  UpdateStockLocationNextInput,
+} from "@medusajs/types"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+import {
+  convertItemResponseToUpdateRequest,
+  getSelectsAndRelationsFromObjectArray,
+} from "@medusajs/utils"
+
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { UpdateStockLocationInput } from "@medusajs/types"
+
+interface StepInput {
+  selector: FilterableStockLocationProps
```

Comment:
Q: shouldn't `FilterableStockLocationProps` requires some cleanup? At the moment, the filterable props are completely different from the other modules, and the comparisons operators differ from the rest of the modules